tasks/sign-index-image/tests/test-sign-index-image-plr.yaml

```diff
@@ -0,0 +1,163 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-sign-index-image-plr
+spec:
+  description: Test creating a internal request to sign an image using the simple-signing-pipeline request
```

Comment:
Wrong description - this is actually testing internal-pipelinerun, not internal request.

Reply:
fixed